File: src/test/java/com/tistory/aristatait/web/dto/HelloResponseDtoTest.java
package com.tistory.aristatait.web.dto;

import org.assertj.core.api.Assertions;
import org.junit.Test;

public class HelloResponseDtoTest {

    @Test
    public void 롬복_기능_테스트() {
        //given
        String name = "test";
        int age = 10;

        //when
        HelloResponseDto dto = new HelloResponseDto(name, age);

        //then
        Assertions.assertThat(dto.getName()).isEqualTo(name);
        Assertions.assertThat(dto.getAge()).isEqualTo(age);

    }
}

File: src/test/kotlin/com/earnestgem/andybook/web/dto/HelloResponseDtoTest.kt
package com.earnestgem.andybook.web.dto

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class HelloResponseDtoTest {

    @Test
    fun test_dataClass() {
        val name = "test"
        val amount = 1000

        val dto = HelloResponseDto(name, amount)

        Assertions.assertThat(dto.name).isEqualTo(name)
        Assertions.assertThat(dto.amount).isEqualTo(amount)
    }
}